src/test/webServer.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-console */
+import { ChildProcess, exec } from 'child_process';
+import net from 'net';
+import os from 'os';
+import stream from 'stream';
+import kill from 'tree-kill';
+import { monotonicTime, raceAgainstDeadline } from './util';
+import { WebServerConfig } from '../../types/test';
+import { assert } from '../utils/utils';
+
+const DEFAULT_ENVIRONMENT_VARIABLES = {
+  'BROWSER': 'none', // Disable that create-react-app will open the page in the browser
+};
+
+const newProcessLogPrefixer = () => new stream.Transform({
+  transform(this: stream.Transform, chunk: any, encoding: string, callback: stream.TransformCallback) {
```

Comment:
Do you have to use `encoding` when doing `toString`?

Reply:
> If the chunk is a string, then this is the encoding type. If chunk is a buffer, then this is the special value 'buffer'. Ignore it in that case.

So I think no, since its guaranteed to be a Buffer and not a string.